### ISSUE TITLE
Log spam

### ISSUE THREAD

**Issue by ChromaPIE** [Red-Studio-Ragnarok/Alfheim#22]
## Describe the bug

```
[16:14:36] [Client thread/INFO] [Alfheim]: 2410.75
[16:14:37] [Client thread/INFO] [Alfheim]: 2048.5
[16:14:37] [Client thread/INFO] [Alfheim]: 2048.5
[16:14:37] [Client thread/INFO] [Alfheim]: 2048.5
[16:14:37] [Client thread/INFO] [Alfheim]: 2048.5
[16:14:37] [Client thread/INFO] [Alfheim]: 2049.5
[16:14:37] [Client thread/INFO] [Alfheim]: 2050.25
[16:14:37] [Client thread/INFO] [Alfheim]: 2049.5
[16:14:37] [Client thread/INFO] [Alfheim]: 2048.75
[16:14:38] [Client thread/INFO] [Alfheim]: 2048.5
[16:14:38] [Client thread/INFO] [Alfheim]: 2049.25
[16:14:38] [Client thread/INFO] [Alfheim]: 2048.5
[16:14:38] [Client thread/INFO] [Alfheim]: 2049.25
[16:14:38] [Client thread/INFO] [Alfheim]: 2048.75
[16:14:38] [Client thread/INFO] [Alfheim]: 2048.25
[16:14:38] [Client thread/INFO] [Alfheim]: 2409.25
[16:14:38] [Client thread/INFO] [Alfheim]: 2048.75
[16:14:38] [Client thread/INFO] [Alfheim]: 2049.0
[16:14:38] [Client thread/INFO] [Alfheim]: 2048.25
[16:14:38] [Client thread/INFO] [Alfheim]: 2048.75
[16:14:38] [Client thread/INFO] [Alfheim]: 2048.75
[16:14:39] [Client thread/INFO] [Alfheim]: 2048.5
[16:14:39] [Client thread/INFO] [Alfheim]: 2048.5
[16:14:39] [Client thread/INFO] [Alfheim]: 2048.5
```

Using Alfheim-1.1
Part of thousands of lines of the log spam
Let me know if more info is necessarily required

**Comment by Desoroxxx** [comment]
OMG, DID I FORREAL, I AM SORRY WILL FIX THIS VERY FAST

**Comment by Desoroxxx** [comment]
Fixed in 1.1.1

- [CurseForge](https://www.curseforge.com/minecraft/mc-mods/alfheim-lighting-engine/files/4881921)
- [Modrinth](https://modrinth.com/mod/alfheim-lighting-engine/version/1.1.1)

**Comment by JL2210** [comment]
lmao just noticed this, thanks
